packages/engine-test-utils/src/__tests__/engine-server/markdown/v5/__snapshots__/hierarchies.spec.ts.snap

```diff
@@ -7,7 +7,7 @@ VFile {
 <hr>
 <strong>Children</strong>
 <ol>
-<li><a href=\\"/notes/foo.ch1.html\\">Ch1</a></li>
+<li><a href=\\"/notes/foo.ch1\\">Ch1</a></li>
```

Comment:
this is actually a change in behavior (albeit you could argue the previous behavior was incorrect). this will affect how child links show up inside the dendron plugin preview.

some rough docs in [[On Click|dendron://dendron.docs/pkg.dendron-plugin-views.ref.note-preview.lifecycle.on-click]]

can you add a test in [[../packages/plugin-core/src/test/suite-integ/PreviewLinkHandler.test.ts]] to make sure this still works on links without the html suffix? (it should)

Reply:
to elaborate, in v5, we introduced `enablePrettyLinks` which strips the `.html` suffix. we set this by default in the `NextJsExportPod`. in this change, since we are making this the default in the dendron config, the snapshot tests are changing because the HTML we generate for the plugin preview now no longer has the `html` suffix. this should be okay - the main behavior this will affect is link handling. we do want to have tests to make sure that this is the case though

Reply:
looks like the tests in `PreviewLinkHandler.test.ts` is already testing for links without the html suffix.